src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -61,7 +61,13 @@ const ParagraphBlock = ({ title, description, button }) => {
 ParagraphBlock.propTypes = {
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
-  button: PropTypes.shape(Button.propTypes),
+  button: PropTypes.arrayOf({
```

Comment:
I think you accidentally changed this from `PropTypes.shape()` to `PropTypes.arrayOf()`. You'll need to be careful and continue to use `.shape()` in the places that had `.shape()` before and `.arrayOf()` in the places that had `.arrayOf()` before.